### DISCUSSION TITLE
Pull Request #95 made the library require PHP ^8.0

### DISCUSSION THREAD

**Discussion by morloderex** [spawnia/sailor#96] [General]
Hey everybody.

Recently https://github.com/spawnia/sailor/pull/95 was merged. But the [mixed type in PHP is a 8.0](https://www.php.net/manual/en/language.types.mixed.php) only feature and the pr is therefore a breaking change that should have been thought of before getting merged.

So what course of action should be taken? Should the pr be reverted in order to maintain PHP 7.4 support. As I know @spawnia have stated [here](https://github.com/spawnia/sailor/pull/89#issuecomment-1458753817) that he planned to support PHP 7.4 for a long time.

My suggestion would be to just remove PHP 7.4 support completely in a new major release probably 1.0 or something and call this library somewhat stable after doing this?

Thoughts?

**Comment by spawnia** [comment]
The pull request you mention only causes type `mixed` to be used in PHPDocs, not in actual type hints. We also have tests proving everything still works with PHP 7.4.
